lib/pbkit/pbkit.c

```diff
@@ -2297,7 +2297,7 @@ void pb_end(DWORD *pEnd)
 }
 
 
-void pb_pushto(DWORD subchannel, DWORD *p, DWORD command, DWORD nparam)
+inline DWORD *pb_pushto(DWORD subchannel, DWORD *p, DWORD command, DWORD nparam)
```

Comment:
Considering https://github.com/XboxDev/nxdk/pull/158/commits/2bcb78718f251614cb68873ec1e33debd46efe1d#r330211066, I'd suggest to make this return `void` for now?

At least it can't be misused then.

Reply:
Fixed - they both return `void` now.